cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -268,10 +269,11 @@ runProjectBuildPhase verbosity ProjectBuildContext {..} =
 selectTargets :: Verbosity -> PackageTarget
               -> (ComponentTarget -> PackageTarget)
               -> [UserBuildTarget]
+              -> Bool
```

Comment:
I'd use a special-purpose `OnlyDependencies` ADT instead of a Bool, but that's a very minor thing.